src/interface/src/styleguide/scenario-card/scenario-card.component.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  Component,
+  Input,
+  HostBinding,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+import {
+  DatePipe,
+  CurrencyPipe,
+  NgIf,
+  NgSwitch,
+  NgClass,
+} from '@angular/common';
+import {
+  StatusChipComponent,
+  StatusChipStatus,
+} from '../status-chip/status-chip.component';
+import { ButtonComponent } from '../button/button.component';
+import { MatIconModule } from '@angular/material/icon';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatButtonModule } from '@angular/material/button';
+export type ScenarioStatus = 'InProgress' | 'Running' | 'Done' | 'Failed';
+/**
+ * Scenario Card for displaying scenario data in a results list
+ */
+@Component({
+  selector: 'sg-scenario-card',
+  standalone: true,
+  imports: [
+    DatePipe,
+    NgIf,
+    CurrencyPipe,
+    NgSwitch,
+    NgClass,
+    StatusChipComponent,
+    ButtonComponent,
+    MatIconModule,
+    MatMenuModule,
+    MatButtonModule,
+  ],
+  templateUrl: './scenario-card.component.html',
+  styleUrl: './scenario-card.component.scss',
+})
+export class ScenarioCardComponent {
+  //TODO: maybe we need to accept the ID here, so we can emit events for these scenarios?
+  @Input() status: ScenarioStatus = 'Running';
+  @Input() name = '';
+  @Input() areas = 0;
+  @Input() budget = 0;
+  @Input() treatmentPlansCount = 0;
+  @Input() creator = '';
+  @Input() created_at = '';
+
+  @Output() openScenarioEvent = new EventEmitter<number>();
```

Comment:
I don't think we need to add `event` to each one of these. Additionally, the emitter is not emitting numbers so you can remove the annotation